frontend/src/components/FileFeedbackPanel.css

```
@@ -0,0 +1,173 @@
+}
+
+.file-feedback__message-text {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  color: var(--text-muted, #888);
+}
+
+.file-feedback__state,
+.file-feedback__anchor {
+  border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)